src/dialects/sqlite/query-generator.js

```diff
@@ -60,13 +60,26 @@ export class SqliteQueryGenerator extends MySqlQueryGenerator {
     let attrStr = attrArray.join(', ');
     const pkString = primaryKeys.map(pk => this.quoteIdentifier(pk)).join(', ');
 
-    if (options.uniqueKeys) {
-      _.each(options.uniqueKeys, columns => {
-        if (columns.customIndex) {
-          attrStr += `, UNIQUE (${columns.fields.map(field => this.quoteIdentifier(field)).join(', ')})`;
-        }
-      });
-    }
+    // sqlite has a bug where using CONSTRAINT constraint_name UNIQUE during CREATE TABLE
+    //  does not respect the provided constraint name
+    //  and uses sqlite_autoindex_ as the name of the constraint instead.
+    //  CREATE UNIQUE INDEX does not have this issue, so we're using that instead
+    //
+    // if (options.uniqueKeys) {
```

Comment:
What are we doing with this code?

Reply:
I'm typically against keeping dead code but I'm hesitant about deleting it, and keep it as a reminder that we should monitor every now and then whether the bug has been fixed in sqlite, and the uncomment this

Reply:
Is there somewhere that we can track this bug?

Reply:
I looked on [sqlite's issue tracker](https://www.sqlite.org/src/wiki?name=Bug+Reports) but couldn't find anything related to this

Reply:
Let's keep this in for now and try to make a bug report for sqlite at a later stage